.github/workflows/gradle-build-pr.yml

```diff
@@ -1,10 +1,12 @@
 name: Integration test and Connected tests
 env:
-  K8_CLUSTER: "integration-master"
+  K8_CLUSTER: 'integration-master'
   # This must be defined for the bash redirection
   GOOGLE_APPLICATION_CREDENTIALS: 'jade-dev-account.json'
   # This must be defined for the bash redirection
   GOOGLE_SA_CERT: 'jade-dev-account.pem'
+  # This will make test logs not use JSON
+  TDR_LOG_APPENDER: 'Console-Standard'
```

Comment:
why does it need to be in single quotes here, but not in the other two yamls?

Reply:
No reason other than I wanted to keep things consistent within the file